python/tvm/relay/op/strategy/mali.py

```diff
@@ -69,6 +71,36 @@ def conv2d_strategy_mali(attrs, inputs, out_type, target):
                 raise RuntimeError(
                     "Unsupported weight layout {} for conv2d NCHW".format(kernel_layout)
                 )
+        elif layout == "NHWC":
+            assert kernel_layout == "HWIO"
+            if not is_auto_scheduler_enabled():
+                logger.error("conv2d NHWC layout is not enabled for mali with autotvm.")
```

Comment:
The message might be a bit confusing for new users, because if they will see this message when building the model even without AutoTVM tuning. In addition, since logging error won't terminate the execution, it will continue and error out when creating the schedule using `naive_schedule`. It seems to me that we should either change this level to warning to let users know the failure reason, or simply raise an exception here.

```suggestion
                logger.error("conv2d NHWC layout is not enabled for mali without auto_scheduler.")
```